components/web-components/select/style.ts

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components'
+
+export const SelectBox = styled.section`
```

Comment:
Is `section` is the right semantic choice here? It will never have any content in it apart from a single `<select` element, and no text / titles / etc.

> The <section> [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML) element represents a generic standalone section of a document, which doesn't have a more specific semantic element to represent it. Sections should always have a heading, with very few exceptions.